<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LegControlWidget</class>
 <widget class="QWidget" name="LegControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="SpinSlider" name="xPositionInput" native="true"/>
   </item>
   <item row="7" column="1">
    <widget class="SpinSlider" name="yPositionInput" native="true"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Z Position</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="calibrateButton">
     <property name="text">
      <string>Calibrate</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="SpinSlider" name="zPositionInput" native="true"/>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="disableButton">
     <property name="text">
      <string>Disable</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Y Position</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="enableButton">
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>X Position</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="allocateButton">
     <property name="text">
      <string>Allocate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Joints</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="JointListItem" name="hipYawJoint" native="true"/>
      </item>
      <item>
       <widget class="JointListItem" name="hipPitchJoint" native="true"/>
      </item>
      <item>
       <widget class="JointListItem" name="kneePitchJoint" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinSlider</class>
   <extends>QWidget</extends>
   <header>spinslider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JointListItem</class>
   <extends>QWidget</extends>
   <header>jointlistitem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
